contracts/implementation/PoolSwapLibrary.sol

```diff
@@ -14,6 +14,17 @@ library PoolSwapLibrary {
 
     uint256 public constant WAD_PRECISION = 10**18;
 
+    struct UpdateData {
+        bytes16 longPrice;
+        bytes16 shortPrice;
```

Comment:
Same again; why `bytes16` here?

Reply:
This is the same reason above. If it were to be stored as `uint256`, we'd just have to convert it back to `bytes16` again when using it in calculations.